tools/docgen.php

```diff
@@ -62,10 +62,11 @@
 
 DOC;
 
-$directories = array_map('basename', glob('../src'.'/*', GLOB_ONLYDIR));
+$providersDir = realpath(__DIR__ . '/../src');
+$directories = array_map('basename', glob($providersDir . '/*', GLOB_ONLYDIR));
 
 foreach ($directories as $provider) {
-    $path = sprintf('%s/../src/%s/README.md', __DIR__, $provider);
```

Comment:
Was there a reason for this change?

Reply:
Yes, which this you can run the docgen file from any directory. For example,

```console
php ../../tools/docgen.php
```